docs/pages/blog.mdx

```diff
@@ -6,24 +6,30 @@ type: posts
 import { getPagesUnderRoute } from "nextra/context";
 import Link from "next/link";
 
-export function BlogIndex({ more = "Read more" }) {
+export function BlogHeader() {
+  return (
+    <div className="max-w-screen-lg mx-auto pt-4 pb-8 mb-16 border-b border-gray-400 border-opacity-20">
+      <h1>
+        <span className="font-bold leading-tight lg:text-5xl">Blog</span>
+      </h1>
+      <p className="text-center text-gray-500 dark:text-gray-400 font-space-grotesk">
+        The latest updates and releases from the Turbo team at Vercel.
+      </p>
+    </div>
+  );
+}
```

Comment:
This change actually fixes one of the suspense boundary issues. The `div` here gets wrapped in a paragraph tag because of `mdx`.